React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -54,6 +54,15 @@ - (void)reactSetFrame:(CGRect)frame
            horizontalScrollerHeight:verticalScrollerWidth];
     [[[scrollView bridge] uiManager] setLocalData:localData forView:self];
   }
+    
+  NSMutableArray *subViews = [[NSMutableArray alloc] init];
+  for (NSView *view in [self subviews]) {
+      if ([view isKindOfClass:[RCTView class]]) {
+        [subViews addObject:view];
+      }
+  }
+
+  [platformScrollView setAccessibilityRows:subViews];
```

Comment:
This would set it for all scrollviews, not just Flatlists. Would that be valid?

Reply:
Good call out! Let me investigate how other scrollviews behavior with this change

Reply:
So I looked at other scrollview examples in RNTester, this is my observation.

https://user-images.githubusercontent.com/67026167/183968272-f1c964bd-0c9f-47f3-ba7f-d62785fd54cb.mov

When navigating through direct subviews inside a scrollview, VO reads out the index correctly. However, when navigating through views inside the subviews, the index is the same for all views.

Reply:
Turns out we don't want every scrollview to behave like a table view, so I'm setting the accessibility role in Flatlist instead :)

Reply:
If I understand correctly, setting `accessibilityRows` in ScrollView is ok because it only makes a difference in VO if the role is `table`? And that is only set when the scrollview is a flatlist?

Reply:
That's my understanding, the documentation states that accessibilityRows only returns the accessible elements for table or outline. https://developer.apple.com/documentation/appkit/nsaccessibilityprotocol/1526702-accessibilityrows

> And that is only set when the scrollview is a flatlist? 
No, currently it's set for every scrollview instance.

Reply:
Sorry I missed this comment. Agreed, I'm going to add a check for that